File: Domain/Model/OrderModel.cs
using Domain.Enum;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Model
{
    public class OrderModel
    {
        public Guid OrderId { get; set; }   
        public string FullName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string ZipCode { get; set; }
        public decimal TotalPrice { get; set; }
        public DateTime CreatedAt { get; set; }

        public PaymentMethod PaymentMethod { get; set; }

        public List<OrderItemModel> OrderItems { get; set; }
    }
}

File: front/src/Interfaces/OrderModel.ts
import { OrderItemModel } from "./OrderItemModel";
import { PaymentMethod } from "./PaymentMethod";

export interface OrderModel {
  orderId: string|null;      // Guid → string
  fullName: string|null;
  email: string|null;
  phone: string|null;
  address: string|null;
  city: string|null;
  zipCode: string|null;
  totalPrice: number|null;   // decimal → number
  createdAt: string|null;    // DateTime → string (ose Date nqs e pars-on)
  paymentMethod: PaymentMethod;
  orderItems: OrderItemModel[];
}